src/python/pants/notes/2.1.x.rst

```diff
@@ -0,0 +1,92 @@
+2.1.x Stable Releases
+=====================
+
+This document describes releases leading up to the ``2.1.x`` ``stable`` series.
+
+See https://www.pantsbuild.org/v2.0/docs/release-notes-2-1 for an overview of the changes in this release, and
```

Comment:
```suggestion
See https://www.pantsbuild.org/v2.1/docs/release-notes-2-1 for an overview of the changes in this release, and
```

The release notes will only be added to the new v2.1 branch of the site.

Reply:
Also the `, and` part is bad. Did you mean to include the link to the plugin upgrade guide?